src/core/gui/XournalView.cpp

```diff
@@ -327,7 +332,8 @@ auto XournalView::onKeyReleaseEvent(const KeyEvent& event) -> bool {
 void XournalView::onRealized(GtkWidget* widget, XournalView* view) {
     // Disable event compression
     if (gtk_widget_get_realized(view->getWidget())) {
-        gdk_window_set_event_compression(gtk_widget_get_window(view->getWidget()), false);
+        g_warning("Find replacement for gdk_window_set_event_compression()");
+        // gdk_window_set_event_compression(gtk_widget_get_window(view->getWidget()), false);
```

Comment:
No need, is active by default and can't be turned off

> With GTK4 one needs to look at the event history between motion events to reconstruct the ones that where coalesced. In gtk4 event delivery rate is fixed to run by the frame clock (that is, 60hz). get_motion_history() (function name may have changed) will only return the motion coordinates that were accumulated during the handling of that frame.

Reply:
> Yes, but we'd want to turn it off... Or at least it is turned off on master.

Oh right, overlooked the false. But using gdk_event_get_history should do the trick.